modules/ROOT/pages/mule-upgrade-tool.adoc

```diff
@@ -0,0 +1,353 @@
+= Mule Upgrade Tool
+
+*Latest version: 1.0.0*
+
+include::release-notes::partial$mule-upgrade-tool/mule-upgrade-tool.adoc[tag=intro]
+
+The Mule upgrade tool runs in any environment that is capable of running a standalone Mule instance. This tool can update only standalone Mule instances that are running any Mule 4 version.
+//You can upgrade local Mule instances to the latest available version or to a particular Mule distribution that you have previously downloaded and unpacked.
+You can also use the Mule upgrade tool to update Mule instances located in different folders.
+
+As part of the upgrade process, the Mule upgrade tool also updates the Runtime Manager agent version if it is installed in your current Mule instance.
```

Comment:
Note: 
1. Mule Runtime Upgrade tool is only available for Mule Runtime version 4 and above. 
2. If Runtime Manager agent is installed in your current Mule instance, Mule upgrade tool will upgrade the Runtime Manager agent version also. 
3. For clustered instances of Mule Runtime, you need to upgrade each cluster nodes independently by following the instructions in xref:release-notes::mule-runtime/updating-mule-4-versions.adoc#mulerunvers[Upgrading an On-Premises Mule Instance Managed Through Runtime Manager]